internal/pkg/cli/task_run.go

```diff
@@ -316,7 +354,7 @@ func (o *runTaskOpts) runTask() ([]*task.Task, error) {
 		o.spinner.Stop(log.Serrorf("Failed to run %s.\n", o.groupName))
 		return nil, fmt.Errorf("run task %s: %w", o.groupName, err)
 	}
-	o.spinner.Stop(log.Ssuccessf("%s %s %s running.\n", english.PluralWord(o.count, "task", ""), o.groupName, english.Plural(o.count, "is", "are")))
+	o.spinner.Stop(log.Ssuccessf("%s %s %s running.\n", english.PluralWord(o.count, "task", ""), o.groupName, english.PluralWord(o.count, "is", "are")))
```

Comment:
It will output "tasks xyz are running" where xyz is the group name. Do you think it makes sense to output "task in group xyz is running" for singular case, and "all tasks in group xyz are running" for the plural case?